app/ui.js

```diff
@@ -197,7 +197,7 @@ var UI;
             UI.initSetting('host', window.location.hostname);
             UI.initSetting('port', port);
             UI.initSetting('encrypt', (window.location.protocol === "https:"));
-            UI.initSetting('true_color', true);
+            UI.initSetting('convertColor', false);
```

Comment:
Other settings such as view only uses underscore_separation instead of camelCase